### ISSUE TITLE
d/s and kB/s d are incorrectly labeled

### ISSUE THREAD

**Issue by jimsalterjrs** [asomers/ztop#47]
I was confused by the undocumented columns "d/s" and "kB/s d", which appear to be intended to track deletions per second (ops) and deletions per second (volume of data).

Upon reading the code, I see that they track nunlinks and nunlinked, respectively. Nunlinks is the kstat which tracks deletion requests queued but not yet completed; nunlinked is the kstat which tracks deletion operations completed.

As things stand, both columns are mislabeled and confusing. Most "normal people" would probably consider both nunlinks and nunlinked as "deletions" if not more closely specified, and there's no way AFAIK to track deletions by the sizes of the files or blocks deleted.

Not sure if the desired fix is to combine nunlinks and nunlinked in a single "d/s" column, or whether you want to just relabel them more appropriately to disclose that both are ops/sec, but one is requests in queue and the other is completed ops.

**Comment by asomers** [comment]
Yes, that's absolutely correct.  I found the same thing myself when I read the source code (long after I initially wrote ztop).  But I haven't got around to fixing it yet.  I think that I might hide both columns behind an off-by-default option.  Or maybe hide nunlinks entirely, because it's so very specific.
